pyswmm/links.py

```diff
@@ -94,14 +94,16 @@ def __getitem__(self, linkid):
     def __iter__(self):
         return self
 
-    def next(self):
+    def __next__(self):
         if self._cuindex < self._nLinks:
             linkobject = Link(self._model, self._linkid)
             self._cuindex += 1  # Next Iteration
             return linkobject
         else:
             raise StopIteration()
 
+    next = __next__  # Python 2
```

Comment:
I mean remove those redefinitions and use directly `six.next` does not that work?